server/api/servarr/base.ts

```diff
@@ -81,6 +110,22 @@ class ServarrBase<QueueItemAppendT> extends ExternalAPI {
     this.apiName = apiName;
   }
 
+  public getSystemStatus = async (): Promise<SystemStatus> => {
+    try {
+      const data = await this.getRolling<SystemStatus>(
```

Comment:
I think it's best we don't use the rolling cache here. Instead, just do a direct request every time so we always have the latest status information.

We use rolling in other places primarily because we don't want to slow down showing profiles/root folders every time we open the request modal.

Reply:
Noted 👍